test/unit/gemm/device/sm90_gemm_f16_f16_f16_tensor_op_f32_tensor_broadcast.cu

```diff
@@ -292,6 +292,253 @@ TEST(SM90_Device_Gemm_f16t_f16t_f32n_tensor_op_gmma_f32_warpspecialized_tensor_b
   EXPECT_TRUE(test::gemm::device::TestAllTensorBroadcast<Gemm>());
 }
 
+/////////////////////////////////////////////////////////////////////////////////////////////////
+///////////////////////////////////// per-column bias ///////////////////////////////////////////
+/////////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
It seems like this PR adds versions of all of the original tests, but with `PerColBias = true`. Would it be possible to reduce the number of tests added so that test count in these files doesn't double?

Reply:
Thank you; yeah I just figured I'd be thorough, but most are just unnecessary. I removed everything but the single fp32 test, and applied the rest of the suggestions.